extensions/wsc/src/test/java/org/mule/extension/ws/metadata/OutputMetadataTestCase.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.ws.metadata;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.isIn;
+import static org.junit.Assert.assertThat;
+import static org.mule.extension.ws.WscTestUtils.ECHO;
+import static org.mule.extension.ws.WscTestUtils.ECHO_ACCOUNT;
+import static org.mule.extension.ws.WscTestUtils.ECHO_HEADERS;
+import org.mule.metadata.api.model.DictionaryType;
+import org.mule.metadata.api.model.MetadataType;
+import org.mule.metadata.api.model.NullType;
+import org.mule.metadata.api.model.ObjectFieldType;
+import org.mule.metadata.api.model.ObjectType;
+import org.mule.metadata.api.model.StringType;
+import org.mule.runtime.api.metadata.descriptor.ComponentMetadataDescriptor;
+import org.mule.runtime.api.metadata.resolving.MetadataResult;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.Optional;
+
+import org.junit.Test;
+import ru.yandex.qatools.allure.annotations.Description;
+
+public class OutputMetadataTestCase extends AbstractMetadataTestCase {
+
+  @Test
+  @Description("Checks the Output Body Metadata for an operation that returns a simple string")
+  public void getEchoOutputBodyMetadata() {
+    MetadataResult<ComponentMetadataDescriptor> result = getMetadata(ECHO_FLOW, ECHO);
+    MetadataType type = result.get().getOutputMetadata().get().getPayloadMetadata().get().getType();
+    ObjectType operation = toObjectType(type);
```

Comment:
The `operarion` Object is only used to get the fields, just store the fields instead of asking two times